frontend/src/model/inferencing/InferenceForm.tsx

```diff
@@ -0,0 +1,125 @@
+  onMemoryChange: (value: number) => void
+} & EditableInferenceFormProps
+
+export default function InferenceForm({
+  model,
+  formData,
+  onImageChange,
+  onDescriptionChange,
+  onPortChange,
+  onProcessorTypeChange,
+  onMemoryChange,
+  editable = false,
+  isEdit = false,
+}: InferenceFormProps) {
+  const theme = useTheme()
+  const isReadOnly = useMemo(() => editable && !isEdit, [editable, isEdit])
+
+  const handleMemoryChange = (_event, newValue) => {
+    onMemoryChange(newValue)
+  }
+
+  const handleImageChange = (image: FlattenedModelImage) => {
+    onImageChange(image)
+  }
+
+  const handlePortChange = (event) => {
+    onPortChange(event.target.value)
+  }
+  const handleProcessorTypeChange = (_event, newValue) => {
+    onProcessorTypeChange(newValue)
+  }
+
+  const handleDescriptionChange = (event) => {
+    onDescriptionChange(event.target.value)
+  }
+  return (
+    <Stack spacing={2}>
+      <Typography fontWeight='bold'>
+        Description {!isReadOnly && <span style={{ color: theme.palette.error.main }}>*</span>}
+      </Typography>
+      {isReadOnly ? (
+        <ReadOnlyAnswer value={formData.description} />
+      ) : (
+        <TextField required size='small' value={formData.description} onChange={handleDescriptionChange} />
+      )}
+      <Typography fontWeight='bold' color='primary' fontSize='medium'>
+        Deployment Settings
+        {!isReadOnly && <HelpPopover>These help you configure how your image is deployed within Bailo</HelpPopover>}
+      </Typography>
+      {!isReadOnly && !editable && (
+        <>
+          <Typography fontWeight='bold'>
+            Image
+            {!isReadOnly && <span style={{ color: theme.palette.error.main }}>*</span>}
+          </Typography>
+          <ModelImageList model={model} value={formData.image} onChange={handleImageChange} />
+        </>
+      )}
+      <Typography fontWeight='bold'>
+        Port {!isReadOnly && <span style={{ color: theme.palette.error.main }}>*</span>}
+      </Typography>
+      {isReadOnly ? (
+        <ReadOnlyAnswer value={formData.port.toString()} />
+      ) : (
+        <TextField required size='small' value={formData.port} onChange={handlePortChange} />
+      )}
+      <Typography fontWeight='bold'>
+        Processor Type {!isReadOnly && <span style={{ color: theme.palette.error.main }}>*</span>}
+      </Typography>
+      {isReadOnly ? (
+        <ReadOnlyAnswer value={formData.processorType} />
+      ) : (
+        <ProcessorTypeList value={formData.processorType} onChange={handleProcessorTypeChange} readOnly={false} />
+      )}
+      <Typography fontWeight='bold'>
+        {`Memory : ${formData.memory} GB `}
+        {!isReadOnly && formData.processorType === 'cpu' && <span style={{ color: theme.palette.error.main }}>*</span>}
+      </Typography>
+      {!isReadOnly && (
+        <Slider
+          disabled={formData.processorType !== 'cpu'}
+          size='small'
+          min={0.1}
```

Comment:
Every other available option is a whole number, so 0.1 sticks out as an outlier. I would either set the minimum as 0 or 1 or adjust the steps to allow for decimal values across the entire slider.